vcluster/configure/vcluster-yaml/external/platform/api-key.mdx

```diff
@@ -2,28 +2,262 @@
+
+- **Shared access** (_Recommended_): Create a single Kubernetes Secret containing the access key, and reference it in multiple virtual clusters, even if they are deployed in different namespaces. This approach simplifies management, reduces overhead, and eases credential rotation, making it suitable for most users and production environments. For most scenarios, using a shared access key is the preferred approach due to its simplicity and ease of maintenance.
+
+- **Individual user access**: Create separate Kubernetes Secrets, each with a unique personal access key, for every virtual cluster. This method provides granular access control and allows revocation of access to individual clusters.
+
+
+<br />
+
+<Tabs>
+  <TabItem value="shared" label="Shared access" default>
+    
+  ### Connect a virtual cluster to the platform using a shared access key
+
+  Use a shared access key to authenticate and connect your virtual cluster to the platform.
+  
+  ### Prerequisites
+
+  Before you begin, ensure you have: 
+
+  - A vCluster Platform instance set up and accessible.
+  - Permissions to create and configure an access key in the platform.
+  - Access to create Kubernetes Secrets on the host Kubernetes cluster (where the virtual cluster runs).
+    
+  ### Configure shared access for a virtual cluster
+
+    <Flow>
+      <Step title="Create the access key">
+        
+        Log in to vCluster platform and create an [access key](/platform/next/api/authentication). 
+        
+      </Step>
+
+      <Step title="Assign the vCluster role">
+
+        When creating the access key, you must define the `vcluster` role in the API key YAML specification in the platform console:
+
+        ```yaml title="Role specification" {3-4}
+        spec:
+          scope:
+            roles:
+              - role: vcluster
+            projects:
+              - project: '*'
+          allowLoftCli: true
+        type: User
+        status: {}
+        ```
+
+        :::note
+        Assigning the `vcluster` role is required. Without it, the vCluster cannot authenticate with the platform and connection errors occur.
+        :::
+
+      </Step>
+      <Step title="Create the Secret">
+        
+        Create the Secret in a namespace on the host cluster. This defaults to `vcluster-platform-api-key` if undefined.
+        
+        :::note
+        This namespace does **not** need to match the target namespace used by the virtual cluster, but it must exist on the host where the vCluster is deployed.
+        :::
+        
+        Run the create command, ensuring you replace the placeholders with your specific values:
+
+        - Replace `ACCESS_KEY` with your API key (also referred to as an access key).
+        - Replace `PLATFORM_HOST` with the platform URL.
+        - Replace `SECRET_NAMESPACE` with the namespace where you want to store the Secret.
```

Comment:
I think inevitably someone will ask in which ns it is best to put the secret, so maybe we suggest vcluster-platform one as default?